### ISSUE TITLE
SMART selftest: Cannot execute SMART on device 'None'

### ISSUE THREAD

**Issue by adriajk** [eReuse/device-inventory#7]
Error from SMART while doing the register.
````
[...]
ERROR:root:Processor.speed
ERROR:root:Processor.address
ERROR:root:Cannot execute SMART on device 'None'.
Connecting to server...
ERROR:root:Error copying file '/tmp/R97841Y.json' to server '192.168.1.2'
Please insert a USB to copy the output or press Ctrl+C to omit.
````

**Comment by slamora** [comment]
This is not a bug, but a mistaken message that alerts of a normal behaviour. We should refactor smart code to avoid annoy the user with wrong error messages.

**Comment by adriajk** [comment]
Fixed on version 7.0.3.
